functions/_fzf_search_processes.fish

```diff
@@ -9,7 +9,8 @@ function _fzf_search_processes --description "Search all running processes. Repl
                      --ansi \
                      # first line outputted by ps is a header, so we need to mark it as so
                      --header-lines=1 \
-                     --preview="ps -o '$ps_preview_fmt' -p {1}" \
+                     # 40 is the length from "PID" to "COMMAND" in the header "PID PARENT USER  %CPU RSS_IN_KB START_TIME COMMAND"
```

Comment:
This spacing may not be portable across different implementations of `ps` but we are not going to jump in the rabbit hole and check `ps` variant, like in the original implementation PR 🤷